src/pages/Delivery/Delivery.jsx

```diff
@@ -35,7 +35,7 @@ const Delivery = () => {
 
   return (
     <Layout
-      packageId={packageInformation?.packages[0]?.id}
+      packageId={packageInformation?.packages[0]?.package_code}
```

Comment:
Please change name or decide if is package code or package id

Reply:
changed, as we spoke we will use package code in all the queries